src/service/command-executor.js

```diff
@@ -94,9 +127,10 @@ class KatalonCommandExecutor extends BaseKatalonCommandExecutor {
     super(info);
     this.teamId = info.teamId;
     this.projectId = info.projectId;
+    this.extraFiles = info.extraFiles;
   }
 
-  preExecuteHook(logger, ksProjectPath) {
+  async preExecuteHook(logger, ksProjectPath) {
```

Comment:
Please check the caller of this method to use `await` properly.

Reply:
I use async because I use await in downloadExtraFiles method.